tests/installation/installation_overview.pm

```diff
@@ -26,6 +26,12 @@ sub run() {
     # preserve it for the video
     wait_idle 10;
 
+    # poo#11438 - Low space warning for before migration
+    if (check_var("UPGRADE", "LOW_SPACE")) {
```

Comment:
My desire would be to have the rest like this (assert when we purposefully do a test with low space) but I ALSO like the idea of a check_screen "low-space-warning" then die "space too low"

Because we need to know when the space is too low either because the product space use has dramatically increased or our images are broken :)